packages/devtools_app/test/standalone_ui/ide_shared/property_editor_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
import libraries directly instead of importing devtools_app.dart

Reply:
Oops, done!